source/dpq/attributes.d

```diff
@@ -221,11 +218,18 @@ template relationName(alias R)
 {
    static if (hasUDA!(R, RelationAttribute))
    {
-      enum rName = getUDAs!(R, RelationAttribute)[0].name;
```

Comment:
And this is the reason why `alias relation = RelationAttribute` won't work. If UDA is declared with `@relation` instead of `@relation()`, the UDA will be a type, rather than a value. And compilation will fail with `Error: need this for name of type string`.